includes/class-newspack-blocks.php

```diff
@@ -80,6 +82,24 @@ function( $classes ) {
 		return $content;
 	}
 
+	/**
+	 * Body class.
+	 *
+	 * @param array $classes Array of body class names.
+	 * @return array Modified array of body class names.
+	 */
+	public static function add_body_classes( $classes ) {
+		$block_theme = 'is-block-theme';
+
+		if ( wp_is_block_theme() && is_admin() ) {
+			$classes .= $block_theme;
+		} elseif ( wp_is_block_theme() ) {
+			$classes[] = $block_theme;
+		}
```

Comment:
Simplifying to this worked for me:

```suggestion
		if ( wp_is_block_theme() ) {
			$classes[] = 'is-block-theme';
		}
```

Reply:
Did it apply the `'is-block-theme` to the editor as well? In my test it didn't that's why I had to add `if ( wp_is_block_theme() && is_admin() ) {`

Reply:
It did in my test site:

<img width="1509" alt="Screenshot 2025-04-11 at 12 24 56 PM" src="https://github.com/user-attachments/assets/776215a9-c32b-4e3b-b7af-dff88fec8fc6" />

Reply:
[See](https://github.com/Automattic/newspack-blocks/pull/2101#issuecomment-2797931817)